pkg/virt-handler/virt-chroot/virt-chroot.go

```diff
@@ -28,20 +30,27 @@ import (
 )
 
 const (
-	binaryPath     = "/usr/bin/virt-chroot"
-	mountNamespace = "/proc/1/ns/mnt"
+	binaryPath = "/usr/bin/virt-chroot"
 )
 
+// GetHostNamespacePath - return the parent process
+// this is needed to handle the case kubernetes and kubevirt
+// running inside a container and the mount namespace is different
+// from the host/node.
+func GetHostNamespacePath() string {
+	return fmt.Sprintf("/proc/%d/ns/mnt", os.Getppid())
```

Comment:
Nit: maybe we could do this only once.

Reply:
@xpivarc do you mean to cache the os.Getppid() once? I can do that